docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -1,17 +1,20 @@
+```eval_rst
+.. warning::  `kedro pipeline package/pull` will be deprecated in 0.18.0. We recommend using `kedro micropkg package/pull` instead.
+```
 
 # Micro-packaging
 
-Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+Micro-packaging allows users to share Kedro micro-packages across codebases, organisations and beyond.
 
-## Package a modular pipeline
+## Package a micro-package
 
-You can package a modular pipeline by executing: `kedro pipeline package <pipeline_name>`
+You can package a micro-package by executing: `kedro micropkg package <pipeline_name>`
```

Comment:
Should we change these instances of `pipeline_name` so they sound more generic? e.g.
```suggestion
You can package a micro-package by executing: `kedro micropkg package <micropkg_name>`
```
Comment applies in multiple places.

Reply:
I agree, it would make more sense with what we are trying to go with.